lib/index.js

```diff
@@ -2,13 +2,73 @@
 
 // Load modules
 
-const Http = require('http');
 const Hoek = require('hoek');
 
 
 // Declare internals
 
-const internals = {};
+const internals = {
+    STATUS_CODES: {
+        __proto__: null,
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer "Prototype mutation".

Makes it so it's a real dictionary instead of a generic JS object. Doesn't have things like `hasOwnProperty`.